canonical/ubuntu.com
# Service Description changes for Pro

## Done

- Changed Ubuntu Advantage Service Terms to Ubuntu Pro Service Description (refer to copy doc for more info)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
    - Visit https://ubuntu-com-12097.demos.haus/legal/ubuntu-pro-description
    -  Copy Doc: https://docs.google.com/document/d/15URkXf2vh7bv89n-cWQPeaP2EehSNUnUCoZvGbNTYFA/edit?usp=sharing

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
